GliaWidgets/Theme/Survey/Theme.Survey.OptionButton.swift

```diff
@@ -39,5 +39,32 @@ extension Theme.SurveyStyle {
             self.font = font
             self.accessibility = accessibility
         }
+
+        /// Apply option button from remote configuration
+        public mutating func applyOptionButtonConfiguration(_ optionButton: RemoteConfiguration.OptionButton?) {
+            applyFontConfiguration(optionButton?.font)
+            normalText.applyTextConfiguration(optionButton?.normalText)
+            normalLayer.applyLayerConfiguration(optionButton?.normalLayer)
+            selectedText.applyTextConfiguration(optionButton?.selectedText)
+            selectedLayer.applyLayerConfiguration(optionButton?.selectedLayer)
+            highlightedText.applyTextConfiguration(optionButton?.highlightedText)
+            highlightedLayer.applyLayerConfiguration(optionButton?.highlightedLayer)
+        }
+
+        /// Apply option button title font from remote configuration
+        private mutating func applyFontConfiguration(_ font: RemoteConfiguration.Font?) {
+            if let fontSize = font?.size, let fontStyle = font?.style {
```

Comment:
Will be better to move it to separate static method.
```suggestion
 extension UIColor {
    static func font(size: CGFloat, style: RemoteConfiguration.FontStyle) -> UIFont {
        switch fontStyle {
        case .bold:
            return UIFont.boldSystemFont(ofSize: fontSize)
        case .italic:
            return UIFont.italicSystemFont(ofSize: fontSize)
        case .regular:
            return UIFont.systemFont(ofSize: fontSize)
        case .thin:
            return UIFont.systemFont(ofSize: fontSize, weight: .thin)
        }
    }
}

if let fontSize = font?.size, let fontStyle = font?.style {
    self.font = UIFont(size: fontSize, style: fontStyle)
}

```

Reply:
FIxed